File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/host/exception/DuplicateSlackUrlException.java
package band.gosrock.domain.domains.host.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class DuplicateSlackUrlException extends DuDoongCodeException {
    public static final DuDoongCodeException EXCEPTION = new DuplicateSlackUrlException();

    private DuplicateSlackUrlException() {
        super(HostErrorCode.DUPLICATED_SLACK_URL);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/host/exception/DuplicateSlackUrlException.kt
package band.gosrock.domain.domains.host.exception

import band.gosrock.common.exception.DuDoongCodeException

class DuplicateSlackUrlException private constructor() : DuDoongCodeException(HostErrorCode.DUPLICATED_SLACK_URL) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = DuplicateSlackUrlException()
    }
}
